python/pyspark/ml/clustering.py

```diff
@@ -36,6 +36,14 @@ def clusterCenters(self):
         """Get the cluster centers, represented as a list of NumPy arrays."""
         return [c.toArray() for c in self._call_java("clusterCenters")]
 
+    @since("1.6.0")
+    def computeCost(self, dataset):
+        """
+        Return the K-means cost (sum of squared distances of points to their nearest center)
+        for this model on the given data.
+        """
+        return self._java_obj.computeCost(dataset._jdf)
```

Comment:
Use _call_java instead of accessing the _java_obj  (That seems to be the more common pattern.)